students/JaeKim/lesson04/assignment/basic_operations.py

```diff
@@ -0,0 +1,86 @@
+import logging
+from customers_model import Customer, TestCustomer
+import customers_model as cm
+
+
+def add_customer(customer_id, name, lastname, home_address, phone_number, email_address, status, credit_limit):
+    """
+    :param customer_id:
+    :param name:
+    :param lastname:
+    :param home_address:
+    :param phone_number:
+    :param email_address:
+    :param status:
+    :param credit_limit:
+    :param stage:
+    :return:
+    """
+    try:
+        credit_limit = float(credit_limit)
+    except ValueError as err:
+        logging.error(err)
+        raise
+
+    args = {
+        'customer_id' : customer_id,
+        'first_name' : name,
+        'last_name' : lastname,
+        'home_address' : home_address,
+        'phone_number' : phone_number,
+        'email_address' : email_address,
+        'status' : status,
+        'credit_limit' : credit_limit
+    }
+
+
+    new_customer = Customer.create(**args)
+    new_customer.save()
+
+
+def search_customer(customer_id):
+    """
+    :param customer_id:
+    :param stage:
+    :return:
+    """
+    try:
+        results = Customer.get(Customer.customer_id == customer_id)
+    except Exception:
+        return False
```

Comment:
Remember to log and try to only catch the exception you expect (in this case I think it is peewee.DoesNotExist). If you aren't sure what you are looking for, temporarily get rid of the try block, purposely run bad data, and see what error you get.